index.bs

```diff
@@ -1222,13 +1239,56 @@ dictionary MLSingleInputSupportLimits {
     :: {{MLSupportLimits}} for output operand.
 </dl>
 
+### {{MLContext/destroy()}}  ### {#api-mlcontext-destroy}
+
+The {{MLContext/destroy()}} method can be called to release all resources associated with the context. Any scheduled compute requests will fail.
+
+<details open algorithm>
+<summary>
+    The <dfn method for=MLContext>destroy()</dfn> method steps are:
+</summary>
+    1. If [=this=] [=MLContext/is lost=], then abort these steps.
+    1. For each {{MLGraph}} |graph| where |graph|.{{MLTensor/[[context]]}} equals [=this=]:
+        1. Run the {{MLGraph/destroy()}} method steps for |graph|.
+    1. Run the steps to [=context lost|lose=] [=this=].
+    1. For each {{MLTensor}} |tensor| where |tensor|.{{MLTensor/[[context]]}} equals [=this=]:
+        1. Run the {{MLTensor/destroy()}} method steps for |tensor|.
+</details>
+
+### Errors ### {#api-mlcontext-errors}
+
+When a user agent determines that an {{MLContext}} is no longer available to fulfill requests, it must run the [=context lost=] steps for it.
+
+<details open algorithm>
+<summary>
+To <dfn lt="context lost">lose</dfn> the {{MLContext}} |context|, run these steps:
+</summary>
+    1. Let |global| be |context|'s [=relevant global object=].
+    1. [=Queue an ML task=] with |global| to run these steps:
+        1. Let |info| be a new {{MLContextLostInfo}}.
+        1. Set |info|.{{MLContextLostInfo/message}} to an [=implementation-defined=] value.
+        1. [=Resolve=] |context|.{{MLContext/[[lost]]}} with |info|.
```

Comment:
These steps should also cause graphs and tensors to be destroyed.

Reply:
Agreed - I should move the steps from destroy() here.